apps/src/templates/sectionAssessments/MultipleChoiceSurveyQuestionDialog.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Button from '@cdo/apps/templates/Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import i18n from '@cdo/locale';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import color from '@cdo/apps/util/color';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {multipleChoiceDataPropType} from './assessmentDataShapes';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  },
+  answers: {
+    float: 'left',
+    width: 550
+  },
+  icon: {
+    color: color.level_perfect
+  },
+  iconSpace: {
+    width: 40,
+    float: 'left'
+  },
+  answerBlock: {
+    width: '100%'
+  },
+  answerLetter: {
+    width: 30,
+    float: 'left',
+    fontWeight: 'bold'
+  }
+};
+
+class MultipleChoiceSurveyQuestionDialog extends Component {
+  static propTypes = {
+    isDialogOpen: PropTypes.bool.isRequired,
+    closeDialog: PropTypes.func.isRequired,
+    questionData: multipleChoiceDataPropType.isRequired
+  };
+
+  render() {
+    const {questionData} = this.props;
+
+    // Questions are in markdown format and should not display as plain text in the dialog.
+
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        isOpen={this.props.isDialogOpen}
+        style={styles.dialog}
+        handleClose={this.props.closeDialog}
```

Comment:
NIT (optional cleanup): access `this.props.isDialogOpen` and `this.props.closeDialog` (here and on line 90) the same way you access questionData at the top of the render function. i.e. `const {questionData, isDialogOpen, closeDialog} = this.props;`

Reply:
i admit that i copy/pasted most of this code from `MultipleChoiceDetailsDialog`, and didn't spend any time looking for places to cleanup the copied code. my gut instinct was to factor out the common parts from both components so as not to duplicate code, but i realized that was a slippery slope leading to an audit of how we compose anything using `BaseDialog` so i decided to punt and opt for the quick and dirty solution until i'm more familiar with our UI codebase. since you mark this as "optional cleanup" i'm going to leave it for now, since if i clean it up here i would feel obligated to clean it up in `MultipleChoiceDetailsDialog` as well, and... well, another slippery slope. tl;dr let's put cleaning up our dialog composition code on the TODO list!

Reply:
> my gut instinct was to factor out the common parts from both components so as not to duplicate code

I like your base instincts. I strongly, strongly support taking the extra time to make our changes as high quality as possible